### PR TITLE
MPP-2442 Avoid to break the button string in 2 lines

### PR DESCRIPTION
**Pull request by lloan**
This PR fixes MPP-2442
 

# New feature description

Originally we had planned to change the width of the element, but when thinking about other languages, if this gets translated, it will continue to break. Spoke to Eduardo to see if he had any other suggestions - he suggested a string update and Chris provided the new string.

# Screenshot (if applicable)

image

# How to test

Test by going to the phone page after verifying your number, this is the step before choosing a relay number mask.

# Checklist

- [ ] l10n changes have been submitted to the l10n repository, if any.
- [ ] All acceptance criteria are met.
- [ ] I've added or updated relevant docs in the docs/ directory.
- [ ] All UI revisions follow the [coding standards](https://github.com/mozilla/fx-private-relay/blob/main/docs/coding-standards.md), and use Protocol tokens where applicable (see `/frontend/src/styles/tokens.scss`).
- [ ] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).

### CODE REVIEW

File: frontend/pendingTranslations.ftl [hunk 1]
```diff
@@ -137,6 +137,7 @@ phone-onboarding-step3-code-success-body = You’re now ready to choose your pho
 phone-onboarding-step3-code-success-subhead-title = What’s next?
 phone-onboarding-step3-code-success-subhead-body = Choose your phone number mask and start using { -brand-name-firefox-relay } to protect your true phone number.
 phone-onboarding-step3-code-success-cta = Search for phone number masks
+phone-onboarding-step3-code-success-cta-2 = Search for masks
```
Comment (Vinnl) [inline frontend/pendingTranslations.ftl:140]: Don't forget to submit this to the l10n repo (in the `phones.ftl` file).

**Review by Vinnl — APPROVED** [review]

File: frontend/pendingTranslations.ftl (hunk 1 above)
Comment (maxxcrawford) [reply frontend/pendingTranslations.ftl:140]: PR opened here, since @lloan is out.

Comment (maxxcrawford) [reply frontend/pendingTranslations.ftl:77]: @Vinnl: Is it okay to leave this in pending for the time being?

Comment (Vinnl) [reply frontend/pendingTranslations.ftl:77]: Thank you! And yes, if the string is (eventually) present in both the l10n repo and pending, the former will have precedence.

**Comment by groovecoder** [comment]
Is this out-dated? Should we update and merge it or just close it?

**Pull request merged**